### PR TITLE
[9.x] Remove unneeded uncountable words

### PR DESCRIPTION
**Pull request by judaschwartz**
Those words are now part of the `Doctrine\Inflector` package they were added here https://github.com/doctrine/inflector/commit/23dc0f6a391e6db6427109363ac48192aa14d53e

### CODE REVIEW

File: src/Illuminate/Support/Pluralizer.php [hunk 1]
```diff
@@ -27,12 +27,7 @@ class Pluralizer
      *
      * @var string[]
      */
-    public static $uncountable = [
-        'cattle',
-        'kin',
-        'recommended',
-        'related',
-    ];
+    public static $uncountable = [];
```
Comment (judaschwartz) [inline src/Illuminate/Support/Pluralizer.php:30]:
```suggestion
    public static $uncountable = ['related'];
```

File: src/Illuminate/Support/Pluralizer.php [hunk 2]
```diff
@@ -27,12 +27,7 @@ class Pluralizer
      *
      * @var string[]
      */
-    public static $uncountable = [
-        'cattle',
-        'kin',
-        'recommended',
-        'related',
-    ];
+    public static $uncountable = ['related'];
```
Comment (judaschwartz) [inline src/Illuminate/Support/Pluralizer.php:30]:
```suggestion
    public static $uncountable = [];
```

File: src/Illuminate/Support/Pluralizer.php (hunk 1 above)
Comment (StewSheen) [inline src/Illuminate/Support/Pluralizer.php:30]:
```suggestion
    public static $uncountable = [
        'recommended',
        'related',
     ];
```

Comment (judaschwartz) [inline src/Illuminate/Support/Pluralizer.php:30]:
```suggestion
    public static $uncountable = [
        'recommended',
        'related',
    ];
```

**Comment by taylorotwell** [comment]
Why did you need to add back "recommended" and "related"?

**Comment by judaschwartz** [comment]
> Why did you need to add back "recommended" and "related"?
they were taken back out of the `Doctrine\Inflector` to the repo because they are not nouns this is only supposed to be a list of uncountable nouns

https://github.com/judaschwartz/inflector/commit/d56c678b61e8fb151384a3ad1726df95674fdb9f

**Pull request merged**